src/components/Table/GridTable.tsx

```diff
@@ -297,6 +297,8 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
   // We only use this in as=virtual mode, but keep this here for rowLookup to use
   const virtuosoRef = useRef<VirtuosoHandle | null>(null);
   const tableRef = useRef<HTMLElement>(null);
```

Comment:
Looks like maybe we don't need `tableRef` anymore?

Reply:
Yea, I don't think so... wasn't sure if I should delete it yet though. I guess I can.

Reply:
Alrighty - since you took the time to see if it was okay to delete, I deleted it.

Reply:
:100: :-)